common/client/multi_node.go

```diff
@@ -661,21 +662,25 @@ func (c *multiNode[CHAIN_ID, SEQ, ADDR, BLOCK_HASH, TX, TX_HASH, EVENT, EVENT_OP
 func aggregateTxResults(resultsByCode sendTxErrors) (txResult error, err error) {
 	severeErrors, hasSevereErrors := findFirstIn(resultsByCode, sendTxSevereErrors)
 	successResults, hasSuccess := findFirstIn(resultsByCode, sendTxSuccessfulCodes)
+	var severeErrorsList error
+	for _, severeError := range severeErrors {
+		severeErrorsList = multierr.Append(severeErrorsList, severeError)
+	}
 	if hasSuccess {
 		// We assume that primary node would never report false positive txResult for a transaction.
 		// Thus, if such case occurs it's probably due to misconfiguration or a bug and requires manual intervention.
 		if hasSevereErrors {
-			const errMsg = "found contradictions in nodes replies on SendTransaction: got success and severe error"
+			const errMsg = "found contradictions in nodes replies on SendTransaction: got success and severe error(s): "
 			// return success, since at least 1 node has accepted our broadcasted Tx, and thus it can now be included onchain
-			return successResults[0], fmt.Errorf(errMsg)
+			return successResults[0], multierr.Combine(fmt.Errorf(errMsg), severeErrorsList)
```

Comment:
Should not we wrap `severeErrorsList` instead of combining? In that case instead of getting
`found contradictions in nodes replies on SendTransaction: got success and severe error(s): ; fatal; fatal`, we'll get
`found contradictions in nodes replies on SendTransaction: got success and severe error(s): fatal; fatal`